client/src/panels/panel_declarations/finances/vote_stat/vote_stat_text.yaml

```diff
@@ -171,3 +167,8 @@ dept_estimates:
 gov_estimates:
   en: Government authorities
   fr: Autorisations du Gouvernement
+
+auth_footnote:
+  transform: [handlebars]
+  en: '"Authorized spendings" are technically known as "{{gl_tt "authorities" "AUTH"}}"'
```

Comment:
Similar to actual_spending_footnote,
"Here, "authorized spending" refers to authorities values"